docs/src/reference/mount.md

```diff
@@ -9,968 +9,856 @@ status: enterprise
 
-### File System Consistency
+1.  **Mount the repository:**
+    This command mounts the `datasets/pets/` prefix from the `main` branch of the `image-repo` repository into a new local directory named `./pets`.
 
-Everest mount provides a strong read-after-write consistency model within a single mount point.
-Once a write operation is done, the data is guaranteed to be available for subsequent read operations.
+    ```bash
+    everest mount "lakefs://image-repo/main/datasets/pets/" "./pets"
+    ```
 
-### lakeFS Consistency
+2.  **Explore the data:**
+    You can now use standard filesystem commands to interact with your data. Files are downloaded lazily only when you access their content.
 
-Local changes are reflected in lakeFS only after the changes are **committed**. Until then, the data is not visible to other users.
-If, for example, two users mount the same branch, they will not see each other's changes until they are committed.
+    ```bash
+    # List files - this only fetches metadata
+    ls -l "./pets/dogs/"
 
-### Sync local changes to lakeFS
+    # Find files
+    find ./pets -name "*.small.jpg"
 
-- As part of `commit` and `diff` commands a `sync` operation will upload all the local changes to a temporary write-branch.
+    # Open a file - this triggers a download
+    open -a Preview "./pets/dogs/golden_retrievers/cute.jpg"
+    ```
 
-## Mount Write Mode File System Behavior
+3.  **Unmount the directory:**
+    When you are finished, unmount the directory.
 
-### Functionality Limitations
+    ```bash
+    everest umount "./pets"
+    ```
 
-- Newly created empty directories will not reflect as directory markers in lakeFS.
-- lakeFS allows having 2 path keys that one is a "directory" prefix of the other, for example the following 2 lakeFS keys are valid: `animals/cat.png` and `animals` (empty object) but since a file system cannot contain both a file and a directory of the same name it will lead to an undefined behavior depending on the Filesystem type (e.g., dir and dir/file).
+---
 
-### File System Behavior
+## Core Concepts
 
-#### Not Supported
+Before diving into data operations, it's important to understand how Everest manages performance, consistency, and caching. These concepts apply to both local and Kubernetes deployments.
 
-- Rename is not supported.
-- Temporary files are not supported.
-- Hard/symbolic links are not supported.
-- POSIX file locks (lockf) are not supported.
-- POSIX permissions are not supported - default permissions are given to files and dirs.
-- A deleted file's name cannot be used as a directory type later and the same for opposite types (e.g, Not allowed: touch foo; rm foo; mkdir foo;).
-- Calling remove on a directory type will fail explicitly with an error.
+### Cache Behavior
 
-#### Behavior modified
+Everest uses a local cache to improve performance when accessing files from lakeFS. Understanding how the cache works will help you optimize performance for your specific use case.
```

Comment:
I think that the guide should give explicit guidelines for how to optimize performance. This line makes the reader work harder to try and keep up and understand how to to it

Reply:
This section was added as part of questions to understand better how the cache works.

Your comment is for us adding another section about how to improve performance and keeping this section to understand how the cache works?